documentation/release_notes.rst

```diff
@@ -8,6 +8,52 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.8.0
+===============
+
+New Features
+------------
+
+Fixed Issues
+------------
+
+Known Issues and Limitations
+----------------------------
+New in This Release
+^^^^^^^^^^^^^^^^^^^
+
+Existing Issues
+^^^^^^^^^^^^^^^
+See oneDPL Guide for other `restrictions and known limitations`_.
+
+- ``histogram`` algorithm requires the output value type to be an integral type no larger than 4 bytes
+  when used with an FPGA policy.
+- Compilation issues may be encountered when passing zip iterators to ``exclusive_scan_by_segment`` on Windows. 
+- For ``transform_exclusive_scan`` and ``exclusive_scan`` to run in-place (that is, with the same data
+  used for both input and destination) and with an execution policy of ``unseq`` or ``par_unseq``, 
+  it is required that the provided input and destination iterators are equality comparable.
+  Furthermore, the equality comparison of the input and destination iterator must evaluate to true.
+  If these conditions are not met, the result of these algorithm calls is undefined.
+- ``sort``, ``stable_sort``, ``sort_by_key``, ``stable_sort_by_key``, ``partial_sort_copy`` algorithms
+  may work incorrectly or cause a segmentation fault when used a device execution policy on a CPU device,
+  and built on Linux with Intel® oneAPI DPC++/C++ Compiler and -O0 -g compiler options.
+  To avoid the issue, pass ``-fsycl-device-code-split=per_kernel`` option to the compiler.
```

Comment:
```suggestion
- ``sort``, ``stable_sort``, ``sort_by_key``, ``stable_sort_by_key``, and ``partial_sort_copy`` may work incorrectly or cause a segmentation fault when used with a device execution policy on a CPU device. This issue occurs when the code is built on Linux with the Intel® oneAPI DPC++/C++ Compiler version 2025.0 or earlier, and with the -O0 -g compiler options. To avoid this issue, pass the ``-fsycl-device-code-split=per_kernel`` option to the compiler.
```

That issue was fixed in Intel® oneAPI DPC++/C++ Compiler 2025.1. I slightly changed the wording since the original first sentence was becoming too long. Linking to https://github.com/uxlfoundation/oneDPL/issues/2011.

Reply:
If the issue is fixed in the new compiler version but not the old ones, I would move the note to https://uxlfoundation.github.io/oneDPL/introduction.html#known-limitations, to shorten the release notes.

And please split into several lines of reasonable length.

Reply:
I've moved the entry into the introduction as suggested with Dmitriy's new wording.